autogpt_platform/frontend/src/hooks/useAgentGraph.tsx

```diff
@@ -814,7 +814,7 @@ export default function useAgentGraph(
                   ?.name?.toLowerCase()
                   .replace(/ /g, "_") || "agentexecutorblock"
               : "agentexecutorblock"
-            : sinkNode.data.title.toLowerCase().replace(/ /g, "_")
+            : sinkNode.data.title.toLowerCase().replace(/ /g, "_").split("_")[0]
```

Comment:
What the split is for ?
Will this cause any risk of non smart decision block code to be broken due to the split ?